bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -137,14 +139,19 @@ templatef = h5py.File(args.bank_file, 'r')
 chisq = trigf[args.ifo+'/chisq'][:]
 chisq_dof = trigf[args.ifo+'/chisq_dof'][:]
 rchisq = chisq / (2 * chisq_dof - 2)
+try:
+    sgchisq = trigf[args.ifo+'/sg_chisq'][:]
+except:
+    sgchisq = None
```

Comment:
what will happen if you feed None to the function that calculates the new statistic?  do we prefer to assign it to an array of zeros in case the file doesn't contain any values?